Add notes about multi-node autoscale, and general tidy up (#758)
timescale/docs: cloud/scaling-a-service.md

# Resizing compute and storage in Timescale Cloud
Timescale Cloud allows you to resize compute (CPU/RAM) and storage independently
at any time. This is useful when you need to do something like increasing your
storage capacity, but not your compute size. You can resize compute and storage

> in the Timescale Cloud console for any service, including multi-node
> environments.

Reviewer: ```suggestion
in the Timescale Cloud console for any service, including members of multi-node
clusters.
```

Storage changes are applied with no downtime, and the new storage capacity is
usually available for use within a few seconds.
*   Storage can only be increased in size. You cannot decrease the amount of
    storage capacity your service has available.
*   Storage size changes can only be made once every six hours.
*   Storage can be modified in various increments between 10&nbsp;GB

>     and 16&nbsp;TB.

Reviewer: After first reading this I had a feeling that 10GB and 16TB applies to increments rather than to the size. Maybe needs to be rephrased in a similar way?

```
Storage size can comprise from 10GB up to 16TB and can be changed in various increments.
```

> *   Storage can be modified in various increments between 10&nbsp;GB
>     and 16&nbsp;TB.

Author: ```suggestion
*   Storage can range in size from 10&nbsp;GB
    to 16&nbsp;TB, and can be changed in various incremeents.
```

longer, and in some cases this could create a delay longer than six hours.

Autoscaling can only increase disk size, not decrease it. You can have a disk up
to 16&nbsp;TB in size.

> If you have a Timescale Cloud multi-node environment, you can also use

Reviewer: We usually refer to them as multi-node clusters. But if it's the same across the whole documentation, then I'm ok with environment.

```suggestion
If you have a Timescale Cloud multi-node cluster, you can also use
```

autoscaling. We recommend that you define different scale limits for the access
node and data nodes, not just because they have different workloads, but also

> because access nodes should scale much more slowly than data nodes. Data nodes

Reviewer: When we are talking that they are going to scale more slowly, it makes an impression that the actual scaling operation for them is going to last longer.

```suggestion
because access nodes are less demanding for storage than data nodes. Data nodes
```
